src/apps/ociocheck/main.cpp

```diff
@@ -209,6 +209,21 @@ int main(int argc, const char **argv)
             for(int i=0; i<config->getNumLooks(); ++i)
             {
                 std::cout << config->getLookNameByIndex(i) << std::endl;
+
+                OCIO::ConstLookRcPtr look = config->getLook(config->getLookNameByIndex(i)); 
+
+                OCIO::ConstTransformRcPtr transform = look->getTransform();         
+
+                try
+                {
+                    OCIO::ConstProcessorRcPtr process = config->getProcessor(transform);
```

Comment:
The `Look` could have forward or inverse transform so the test must validate the right one. If `transform` is null then `look->getInverseTransform()` must be used.

Reply:
@hodoulp 

when I am using 
'''
       OCIO::ConstTransformRcPtr  invTransform = look->getInverseTransform();         
'''

it results in Segmentation fault (core dumped). 

I think its because inverse transform is not defined in the config and we are trying to access it causing segmentation fault to occur.

Reply:
On a valid instance, `ConstTransformRcPtr Look::getInverseTransform()` could return an empty [std::smart_ptr](https://en.cppreference.com/w/cpp/memory/shared_ptr) but it should not crash. However, if the code later deferences it than it crashes.

You should use a debugger to see where the problem is coming from.

Reply:
@hodoulp 

Can you tell me how to use debugger in ocio?

Reply:
@YashAgrawal0 

Assuming that you are working on Linux:
1. Compile in 'debug' mode i.e. look at [CMakeLists.txt](https://github.com/AcademySoftwareFoundation/OpenColorIO/blob/master/CMakeLists.txt), and search for `CMAKE_BUILD_TYPE`
2. Use your favorite debugger, to start ociocheck.exe i.e. gdb for example on Linux platform.
>gdb .\src\apps\ociocheck\ociocheck

Lot of documentations are available on the web, but you can start with these gdb commands: [r|run], [n|next], [p|print].